Project-Timer-frontend/src/app/shared/interfaces/project.interface.ts

```diff
@@ -0,0 +1,8 @@
+import { Task } from "./task.interface";
```

Comment:
czemu  ""  a nie  '' ?

Reply:
a to jest jakaś różnica? Standard przyjmuje że jest ''?